test/Main.hs

```diff
@@ -52,7 +53,23 @@ testImplies :: Bit -> Bit -> Expectation
 testImplies a b
   | a == I = implies a b `shouldBe` b
   | otherwise = implies a b `shouldBe` I
-  
+
+comHalfAdder :: Bit -> Bit -> Expectation
+comHalfAdder a b = halfAdder a b `shouldBe` halfAdder b a
```

Comment:
```suggestion
testHalfAdderCommutative :: Bit -> Bit -> Expectation
testHalfAdderCommutative a b = halfAdder a b `shouldBe` halfAdder b a
```

The convention is to prefix with `test`, and I think it's better to write out the property, because it's not immediately obvious what `Com` means without reading the whole thing.